enterprise-suite/tests/smoke_ingress

```diff
@@ -32,8 +32,10 @@ ingress_ready() {
     # Debug
     # kubectl describe ing/ingress-test -n lightbend
     # ingress can take up to 1m to get ready
-    sleep 2
     kubectl describe ing/ingress-test -n lightbend | awk 'END { exit !( $1 == "Normal"  &&  $2 == "UPDATE" ) }'
+    status=$?
+    sleep 2
```

Comment:
why move the sleep?

Reply:
sleeping more after we have the answer doesn't... change the answer or anything else about this function. debug leftover?

Reply:
We have _an_ answer but it seems even that was marginally premature.  With the sleep after, the stated condition appears to be reality.

Reply:
Everything is async in k8s so I can see why this might be necessary, as the status value in the apiserver might not reflect reality (but that would be pretty crap).

Reply:
ah! totally agree on looking for a better condition if this one is not yet correct. busy wait until we can curl the ingress?

Reply:
Yeah, I was looking for a test that only involved kubectl things.  After this test passes, indicating the ingress is ready, we test by curling the ingress.  Seemed a bit silly to only test curling the ingress only after determining it was up by curling the ingress...  But if the test I chose doesn't work then there may be no choice.

Reply:
maybe instead of sleep 2 here you could busy_wait in test_es_console_responding

something somewhere has to keep trying the curl thing until it works, if we think it may be timing sensitive from this step. sleep 2 is just going to work until we needed sleep 3, should busy_wait instead of sleep